src/seraphis_mocks/jamtis_mock_keys.h

```diff
@@ -71,6 +77,21 @@ struct jamtis_mock_keys
     rct::key K_1_base;                //jamtis spend base     = k_vb X + k_m U
     crypto::x25519_pubkey xK_ua;      //unlock-amounts pubkey = xk_ua xG
     crypto::x25519_pubkey xK_fr;      //find-received pubkey  = xk_fr xk_ua xG
+
+    crypto::chacha_iv m_encryption_iv;
+
+    BEGIN_SERIALIZE()
+    FIELD(k_m)
+    FIELD(k_vb)
+    FIELD(xk_ua)
+    FIELD(xk_fr)
+    FIELD(s_ga)
+    FIELD(s_ct)
+    FIELD(K_1_base)
+    FIELD(xK_ua)
+    FIELD(xK_fr)
+    END_SERIALIZE()
+
 };
```

Comment:
```suggestion
    END_SERIALIZE()
};
```

Please clean up whitespace issues like this before making a PR.